terminal_colors/terminal_colors.py

```diff
@@ -554,10 +560,22 @@ def _terminal():
         terminal class
     """
     curses.setupterm()
-    num_colors = curses.tigetnum('colors')
-    return {16:term16, 88:term88, 256:term256}.get(num_colors, term16)
+    if options.force_term is None:
+        num_colors = curses.tigetnum('colors')
+    else:
+        num_colors = options.force_term
+    if num_colors > 0:
+        return TERM_TABLE.get(num_colors, term16)
+
 
+TERM_TABLE = {16:term16, 88:term88, 256:term256}
```

Comment:
TERM_TABLE references the classes `term16`, etc. and so it was easiest to declare TERM_TABLE later in the script, after the classes have been parsed by the Python interpreter.